tests/e2e/e2e_test.go

```diff
@@ -44,28 +42,21 @@ func TestMain(m *testing.M) {
 	// Register test flags, then parse flags.
 	framework.RegisterClusterFlags(flag.CommandLine)
 	flag.Parse()
-	klog.SetOutput(ginkgo.GinkgoWriter)
 
 	// check if flags are set and if not cancel the test run
 	if *ImageRepo == "" || *ImageTag == "" || *HelmChart == "" {
-		e2elog.Failf("Required flags not set. Please set -image.repo, -image.tag and -helm-chart")
+		log.Fatal("Required flags not set. Please set -image.repo, -image.tag and -helm-chart")
 	}
 
 	os.Exit(m.Run())
 }
 
 func TestE2E(t *testing.T) {
-	e2elog.InitLogs()
-	defer e2elog.FlushLogs()
-	klog.EnableContextualLogging(true)
```

Comment:
Do we still enable contextual logging?

Reply:
helm is doing it, and we then leave logging to ginkgo/gomega logger